src/applications/appeals/996/content/InformalConference.jsx

```diff
@@ -1,92 +1,52 @@
 import React from 'react';
 
-import { Toggler } from 'platform/utilities/feature-toggles';
-
-export const informalConferenceTitle = 'Request an informal conference';
-
 export const newInformalConferenceTitle =
```

Comment:
I think we should rename this to the now removed `informalConferenceTitle` as calling it "new" doesn't make sense anymore.

Reply:
Done